src/SOLID/LiskovSubstitutionPrinciple/Error/BranchOffice.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Deg540\PHPTestingBoilerplate\SOLID\LiskovSubstitutionPrinciple\Error;
+
+class BranchOffice
+{
+    /**
+     * @var Developer[]
+     */
+    private array $developers;
+
+    public function insertDevelopers(Developer $developer): void
```

Comment:
¿en singular? Este y en los demás casos. 
```suggestion
    public function insertDeveloper(Developer $developer): void
```

Reply:
Cambiado